pages/settings/audits.tsx

```diff
@@ -1,30 +1,58 @@
 import { type GetServerSidePropsContext } from "next";
+import { Fragment, useState } from "react";
 import SettingsLayout from "@/layouts/Settings";
 import { getServerSideSession } from "@/utils/session";
+import { trpc } from "@/utils/trpc";
 import { User } from "@prisma/client";
+import { useQuery } from "@tanstack/react-query";
 import AuditLogs from "@/components/projects/AuditLogs";
 import { Button } from "@/components/theme";
 import Audit from "@/lib/audit";
 
 interface AuditSettingsProps {
   user: User;
-  logs: any;
 }
 
-const AuditSettings = ({ user, logs }: AuditSettingsProps) => {
+const AuditSettings = ({ user }: AuditSettingsProps) => {
+  const [page, setPage] = useState(0);
+  const {
+    data: auditLogs,
+    fetchNextPage,
+    isLoading,
+    status,
+    isFetchingNextPage,
+    hasNextPage
+  } = trpc.audits.paginate.useInfiniteQuery(
+    { limit: 20 },
```

Comment:
Lets do the limit of `10` so it nicely fits on the page.

Reply:
sure